services/provider/server/server.go

```diff
@@ -910,6 +941,17 @@ func (s *OCSProviderServer) getOCSSubscriptionChannel(ctx context.Context) (stri
 	return subscription.Spec.Channel, nil
 }
 
+func (s *OCSProviderServer) getClientMapping(ctx context.Context) (*corev1.ConfigMap, error) {
```

Comment:
What return a config map and not just `map[string]string`. You are not interested in the resource, just in the mapping.

Also, please rename this func to something more specific